doc/vaulted-edit.1.md

```diff
@@ -21,7 +21,20 @@ Upon quitting, the new content is saved to the vault.
 AWS KEY
 -------
 
+* k - Key  
+   Manages Access Key ID and Secret Access Key credentials.
+* m - MFA  
+   Manages MFA (multi-factor authentication) ARN or serial number. If enabled, user will be prompted
+   to enter MFA code when accessing vault. MFA will remain active for the
+   vault duration.
+* r - Role  
+   ARN of role to be assumed when accessing vault.
 * t - Substitute with temporary credentials  
    Toggles whether your AWS credentials are substituted with a set of temporary
    credentials. For more details on this process, see the documentation for
-   `vaulted shell`.
\ No newline at end of file
+   `vaulted shell`.
+* S - Show/Hide Key  
+   Toggles whether Secret Access Key ID is displayed
+   when viewing vault details in the edit menu. Set to hide by default.
```

Comment:
I think "default" is ambiguous here. Perhaps this should describe how it is a transitory setting that only affects the current editing session and is set to hidden at the start of each editing session.